database_manager/command_handlers/helpers/database_updater.py

```diff
@@ -0,0 +1,123 @@
+            {user_requests_attribute: new_value},
+            synchronize_session='fetch'
+        )
+
+    def insert_values(self,
+                      table: Union[Users, UserRequests, Watches, Mustwatches],
+                      **kwargs: dict) -> Query:
+        record = table(
+            **kwargs
+        )
+        self.session.add(record)
+        self.session.commit()
+        return record
+
+    def update_user_request(self,
+                            user_requests_attribute: UserRequests,
+                            chat_id: str,
+                            user_id: str,
+                            new_value: Union[bool, str, int]) -> None:
+        user = self.get_user_record(chat_id, user_id)
+        self.update_user_request_attribute(user, user_requests_attribute, new_value)
+        self.session.commit()
+
+    def update_user_request_message_id(self,
+                                       chat_id: str,
+                                       user_id: str,
+                                       message_id: str) -> None:
+        self.update_user_request(UserRequests.message_id, chat_id, user_id, message_id)
+
+    def delete_user_score_from_user_request(self,
+                                            chat_id: str,
+                                            user_id: str) -> None:
+        self.update_user_request(UserRequests.user_score, chat_id, user_id, None)
+
+    def update_user_request_add_or_delete(self,
+                                          chat_id: str,
+                                          user_id: str,
+                                          chosen_action_on_mustwatch: str) -> None:
+        if chosen_action_on_mustwatch == btn.ADD_BUTTON_CALLBACK:
+            self.update_user_request(UserRequests.add_or_delete, chat_id, user_id, True)
+        else:
+            self.update_user_request(UserRequests.add_or_delete, chat_id, user_id, False)
+
+    def update_user_request_chosen_user_from_chosen_action(self,
+                                                           chat_id: str,
+                                                           user_id: str,
+                                                           chosen_action_on_mustwatch: str) -> None:
+        if chosen_action_on_mustwatch == btn.DELETE_BUTTON_CALLBACK:
+            self.update_user_request(UserRequests.chosen_user_id, chat_id, user_id, btn.ALL_BUTTON_CALLBACK)
+        elif chosen_action_on_mustwatch == btn.SHOW_MUSTWATCHES_BUTTON_CALLBACK:
+            self.update_user_request(UserRequests.chosen_user_id, chat_id, user_id, btn.ME_BUTTON_CALLBACK)
+
+    def add_mustwatch_to_tables(self,
+                                title: str,
+                                chosen_user_id: str,
+                                chat_id: str) -> None:
+        watch_record = self.get_watch_record(title, chat_id)
+        if not watch_record:
+            self.insert_values(Watches, title=title, group_id=chat_id)
```

Comment:
Please pay attention that this operation is not atomic. You could insert `watch_record`, but then you won't `watches_id`. It leads to inconsistence.